client/my-sites/add-ons/hooks/use-add-ons.ts

```diff
@@ -0,0 +1,86 @@
+import {
+	WPCOM_FEATURES_NO_ADVERTS,
+	WPCOM_FEATURES_CUSTOM_DESIGN,
+	WPCOM_FEATURES_UNLIMITED_THEMES,
+} from '@automattic/calypso-products';
+import { useTranslate } from 'i18n-calypso';
+import { useSelector } from 'react-redux';
+import {
+	getProductBySlug,
+	getProductDescription,
+	getProductDisplayCost,
+	getProductName,
+	getProductTerm,
+} from 'calypso/state/products-list/selectors';
+import customDesignIcon from '../icons/custom-design';
+import noAdsIcon from '../icons/no-ads';
+import unlimitedThemesIcon from '../icons/unlimited-themes';
+
+export interface AddOnMeta {
+	slug: string;
+	highlight?: boolean;
+	icon: JSX.Element;
+	name: string | React.ReactChild | null;
+	description: string | React.ReactChild | null;
+	displayCost: string | React.ReactChild | null;
+	term?: string | React.ReactChild | null; // year, month, etc.
+}
+
+// memoize on products list
+const useAddOns = (): ( AddOnMeta | null )[] => {
+	const translate = useTranslate();
+
+	// list add-ons to show in UI here
+	const addOnsActive = [
+		{
+			slug: WPCOM_FEATURES_NO_ADVERTS,
+			highlight: false,
+			icon: noAdsIcon,
+			overrides: {
+				// override API-fetched metadata
+				name: translate( 'Remove Ads' ),
+			},
+		},
+		{
+			slug: WPCOM_FEATURES_CUSTOM_DESIGN,
+			highlight: false,
+			icon: customDesignIcon,
+			overrides: null,
+		},
+		{
+			slug: WPCOM_FEATURES_UNLIMITED_THEMES,
```

Comment:
We tried adding this feature to the backend, but it already exists under another name/slug. Given that we can have features and products with mismatching slugs, this map will need to be different and have both featureSlug and productSlug.

The feature for themes is WPCOM_FEATURES_PREMIUM_THEMES and the productSlug would be 'unlimited_themes'.